Generators/share/egconfig/pythia8_pO.cfg

```diff
@@ -1,8 +1,10 @@
 ### beams
+Beams:frameType 2       # back-to-back beams of different energies and particles
 Beams:idA 2212          # proton
 Beams:idB 1000080160    # Oxygen
-Beams:eCM 13600.        # GeV
+Beams:eA 6800.          # Energy of proton beam in GeV moving in the +z direction
+Beams:eB 3400.          # Energy of Oxygen beam in GeV moving in the -z direction
```

Comment:
The value is correct, but IMHO the most correct way to represent this would be to define the energy in ZTeV, and then get eB.

Reply:
Unfortunately this does not to seem to be an option in Pythia8 parameters. Only the energy of the beams can be specified

Reply:
thanks a lot, this looks okay - I'm not sure if PYTHIA will take Z TeV but maybe one can just write it into the comments? 

Then this is perhaps a silly question but are we sure eA is proton and eB is oxygen in terms of direction? I do not know myself (we could also have a pO and Op cfg file if we think it's necessary)

Reply:
I edited the comment next to eB energy, it should be clearer now. Regarding the direction, I see that for pPb we assume it's beam A as proton. I would keep it this way and edit it in the future in case.

Reply:
This is all good - IMHO we should proceed and auto-generate (in whichever way you think is best :-) ) the config into O2DPG. Thank you very much!

Reply:
For the beams: beam 1 (from ATLAS to ALICE, clockwise, from A to C side(*)) will be protons, beam 2 (from CMS to ALICE, anticlockwise, from C to A side (*)) will be oxygen. But I don't know what is A and B here.

(*) For the direction within ALICE, see also https://arxiv.org/html/2204.10148v3